main.go

```diff
@@ -14,18 +14,18 @@ import (
 type Import struct {
 	// used in refresh only
 	TokenId   uint   `json:"tokenId" example:"0"`
-	Token     string `json:"token" example:"sample token"`
+	Token     string `json:"token" example:"sample_token"`
 	User      string `json:"user" example:"sample user name"`
-	Url       string `json:"url" example:"https://gitlab.local"`
+	Url       string `json:"url" example:"https://api.github.com/"`
 	UploadUrl string `json:"uploadUrl" example:""`
 	// used in refresh only
 	ProviderId uint `json:"providerId" example:"0"`
 	// azuredevops, gitlab or github
-	Provider string `json:"provider" example:"gitlab"`
+	Provider string `json:"provider" example:"github"`
 	// used in refresh only
 	ProjectId uint   `json:"projectId" example:"0"`
-	Project   string `json:"project" example:"sample project name"`
-	Group     string `json:"group" example:"default"`
+	Project   string `json:"project" example:"sample project name (wharf-api)"`
+	Group     string `json:"group" example:"default (iver-wharf)"`
```

Comment:
Suggest only using 1 example instead of trying to mix two different

```suggestion
	Project   string `json:"project" example:"wharf-provider-github"`
	Group     string `json:"group" example:"iver-wharf"`
```